client.go

```diff
@@ -1060,7 +1095,7 @@ func (c *Client) isSupportedType(offset int, args ...interface{}) error {
 	for idx, arg := range args {
 		switch arg.(type) {
 		case string, int, int64, float64, bool:
-			return nil
+			continue
```

Comment:
nit: I kind of prefer the previous one :thinking:

Reply:
But it doesn't actually work, hence the dedicated test I wrote illustrating it 👍🏻 
If you return, you do not handle all the args, thus if any of them is actually invalid (when handling arrays for instance), you miss it, but redis won't 🙇🏻

Reply:
aaah .... yeah, correct :facepalm: